contrib/externalMatchfinder/main.c

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright (c) Yann Collet, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under both the BSD-style license (found in the
+ * LICENSE file in the root directory of this source tree) and the GPLv2 (found
+ * in the COPYING file in the root directory of this source tree).
+ * You may select, at your option, one of the above-listed licenses.
+ */
+
```

Comment:
Could you add a short description of what this file is about, what are its objectives ?

Reply:
Added contrib/externalMatchfinder/README.md